app/assets/stylesheets/application.scss

```diff
@@ -1,5 +1,9 @@
 @charset "utf-8";
 
+// Cannot use GDS Transport font on a non GOV.UK domain due to licensing
```

Comment:
Is there any way to add this conditionally only on the trial domain?

Reply:
I can check for the host and conditionally add a class name to the body but I don't think its worth the overhead/code bloat....